buildSrc/src/main/kotlin/Dependencies.kt

```diff
@@ -46,14 +47,15 @@ object Libraries {
 object TestLibraries {
     private object Versions {
         const val junit4 = "4.13"
-        const val mockito = "2.7.22"
+        const val mockito = "3.3.0"
         const val robolectric = "4.3.1"
         const val assertJ = "3.16.1"
         const val testRunner = "1.1.0"
         const val espresso = "3.2.0"
         const val testExtensions = "1.1.1"
         const val testRules = "1.1.0"
         const val uiAutomator = "2.2.0"
+        const val coroutines = "1.3.7"
```

Comment:
We might want to consider moving this out of the `TestLibraries.Versions` seeing as we're most definitely need this for the `Libraries` object too (it also looks like you're missing a coroutines dependency there too)

Reply:
Good catch, thanks! They were added as transitive dependencies. I added them explicitly to force the same version

+--- androidx.lifecycle:lifecycle-viewmodel-ktx:2.2.0
|    +--- org.jetbrains.kotlin:kotlin-stdlib:1.3.50 -> 1.3.72 (*)
|    +--- org.jetbrains.kotlinx:kotlinx-coroutines-android:1.3.0
|    |    +--- org.jetbrains.kotlin:kotlin-stdlib:1.3.71 -> 1.3.72 (*)
|    |    \--- org.jetbrains.kotlinx:kotlinx-coroutines-core:1.3.0 (*)